test/amortizer.js

```diff
@@ -7,67 +15,68 @@ var assert     = require('assert');
 describe('Amortizer', function() {
   var token, startTime, startBalance;
 
-  before(function(done) {
-    startTime = Date.now();
-    startBalance = 10;
-    amortizer._instances[token] = {
-      lastCheckedBalance: startBalance,
-      lastChargedTime: startTime
-    }
-    new Token({ token: uuid.v4() }).save().then(function(token_) {
-      token = token_;
+  before(function() {
+    return db.knex.migrate.rollback(db.conf);
+  });
+
+  beforeEach(function() {
+    return db.knex.migrate.latest(db.conf).then(function() {
       startTime = Date.now();
-      amortizer._instances[token.get('token')] = {
-        lastCheckedBalance: 10,
+      startBalance = 10;
+      amortizer._instances[token] = {
+        lastCheckedBalance: startBalance,
         lastChargedTime: startTime
       }
-      token.getBalance().then(function(balance) {
```

Comment:
No idea. I wanted to modify the tests as little as needed so I kept it in there.